tests/tests_pytorch/deprecated_api/test_remove_1-8.py

```diff
@@ -323,48 +322,6 @@ def on_pretrain_routine_end(self, trainer, pl_module):
         trainer.fit(model)
 
 
-@pytest.mark.flaky(reruns=3)
-@pytest.mark.parametrize(["action", "expected"], [("a", [3, 1]), ("b", [2]), ("c", [1])])
-def test_simple_profiler_iterable_durations(tmpdir, action: str, expected: list):
```

Comment:
IMO this test should not be removed but refactored to not use `profile_iterable`

Reply:
understood. Thanks for clarification.